introduction.md

```diff
@@ -1,2 +1,6 @@
 Hello, it's me, Maaike.
-I liked the Javascript course very much.
\ No newline at end of file
+I liked the Javascript course very much.
+
+* Git
+* Javascript
+* Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?